samples/test/support/management-api/createUser.ts

```diff
@@ -37,20 +37,23 @@ export default async (firstName: string, assignToGroup = 'Basic Auth Web'): Prom
     });
     user.credentials.password.value = password;
 
-    // TODO: create test group and attach password recovery policy during test run when API supports it
-    const {value: testGroup} = await oktaClient.listGroups({
-      q: assignToGroup
-    }).next();
-
-    if (!testGroup) {
-      throw new Error(`Group "${assignToGroup}" is not found`);
-    }
-
     await oktaClient.assignUserToApplication(config.clientId as string, {
       id: user.id
     });
+    
+    for (const groupName of assignToGroups) {
```

Comment:
looks like you made change here in both PRs, let's focus on merge one first to avoid merge conflict issues.